************************
Test -> 0-add_integer.py
========================
>>> add_integer = __import__('0-add_integer').add_integer

=======SUCCES CASES============

>>> add_integer(0, 0)
0

>>> add_integer(-5, -5)
-10

>>> add_integer(5, 5)
10

>>> add_integer(10, -2)
8

========ERROR string passed==========
>>> add_integer("num 20", 1002)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(5.403, "random_String")
Traceback (most recent call last):
TypeError: b must be an integer

=========ERROR Empty arguments========
>>> add_integer(None)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(None, 1002)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(103, None)
Traceback (most recent call last):
TypeError: b must be an integer


